### PR TITLE
[11.0][FIX] stock: set use_propagation_minimum_delta to TRUE

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
We set this new parameter to true to maintain the same behavior of v10 (as it should be expected).

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by JordiMForgeFlow — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I don't see where is this used in code.

**Comment by MiquelRForgeFlow** [comment]
This parameter lets set the `propagation_minimum_delta` field. See https://github.com/OCA/OpenUpgrade/blob/11.0/addons/stock/views/res_config_settings_views.xml#L288. The field was visible in v10 because this parameter didn't exist yet.

**Comment by MiquelRForgeFlow** [comment]
We also need this because in the v13 migration, we use this parameter to set the `propagate_date`.

File: addons/stock/migrations/11.0.1.1/post-migration.py [hunk 1]
```diff
@@ -439,6 +439,12 @@ def create_stock_move_line_from_inventory_moves(env):
     )
 
 
+def fill_config_parameter_use_propagation_minimum_delta(env):
+    # new parameter, we set to true to maintain same behavior of v10
```
Comment (pedrobaeza) [inline addons/stock/migrations/11.0.1.1/post-migration.py]: Why don't you put this as docstring?

Comment (MiquelRForgeFlow) [reply addons/stock/migrations/11.0.1.1/post-migration.py]: Why should I? I don't care, but I will do if you care haha

Comment (pedrobaeza) [reply addons/stock/migrations/11.0.1.1/post-migration.py]: Each tool is for each usage. If you are documenting the aim of the method, use docstring. If it's a clarification about specific line/s of code, the use comments.

Comment (MiquelRForgeFlow) [reply addons/stock/migrations/11.0.1.1/post-migration.py]: Ok, done.

**Pull request merged**